src/sentry/utils/data_filters.py

```diff
@@ -0,0 +1,48 @@
+"""
+sentry.utils.data_filters.py
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import fnmatch
+
+
+class FilterTypes(object):
+    ERROR_MESSAGES = 'error_messages'
+    RELEASES = 'releases'
+    BLACKLISTED_IPS = 'blacklisted_ips'
```

Comment:
I'd either change the other (preexisting) code that uses the `blacklisted_ips` magic string to use this value, or not add it here yet and follow back up with a cleanup patch that adds it here and standardizes the existing usage. Your call on which approach makes more sense to you (if it were me, I'd punt on it for now and improve it as a follow up to reduce the scope of this change, but that's just my opinion)